### ISSUE TITLE
20H2 OS Version is not available for Lenovo Models

### ISSUE THREAD

**Issue by jp2490** [maurice-daly/DriverAutomationTool#191]

**Comment by Wavee7** [comment]
On what model are you trying to apply the drivers ?

**Comment by jp2490** [comment]
I am trying to get Lenovo Models for Windows 10 20H2. Despite full internet access, tool is not able to query the catalog site. I think that's why I am not able to select 20H2 in drop down.

 I am using the stable release 6.4.9

 
![image](https://user-images.githubusercontent.com/63575795/108953057-8e257e00-7638-11eb-831a-e18c5019dd45.png)

**Comment by AlkHacNar** [comment]
have you applyed the exe with the lenovo fix, or only run the installer? i had the same at first, but after updating the exe it works fine

**Comment by Wavee7** [comment]
Yes, @AlkHacNar is right, you need to update the tool to the latest version
